cert_munger.gemspec

```diff
@@ -19,15 +19,16 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.required_ruby_version = '>= 1.9.3'
-  spec.add_dependency 'logging', '~> 1.8'
+  spec.add_dependency 'logging', '~> 2.1'
 
   spec.add_development_dependency 'awesome_print', '~> 1.2'
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'codeclimate-test-reporter'
+  spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'rack', '~> 1.5'
   spec.add_development_dependency 'rack-test', '~> 0.6'
   spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rubocop', '~> 0.27'
+  spec.add_development_dependency 'rubocop', '~> 0.41.1'
```

Comment:
Pretty sure this will need to be a hard-lock, version `0.41.2` of Rubocop is what required Ruby 2.0 support I believe.